cmd/config-current.go

```diff
@@ -544,6 +545,10 @@ If you need help to migrate smoothly visit: https://min.io/pricing`
 	}
 
 	for k, l := range loggerCfg.Audit {
+		if (l.ClientCert == "" && l.ClientKey != "") ||
+			(l.ClientCert != "" && l.ClientKey == "") {
+			logger.LogIf(ctx, fmt.Errorf("Unable to initialize audit HTTP target: %w", errors.New("cert and key must be specified as a pair")))
+		}
```

Comment:
This code should be validated only if l.Enabled is true @nitisht so this should be validated when the loggerCfg.Audit does the lookup from config.